resources/event-bus/charts/event-publish-service/templates/service-monitor.yaml

```diff
@@ -16,5 +16,9 @@ spec:
 {{- if .Values.monitoring.scrapeInterval }}
     interval: {{ .Values.monitoring.scrapeInterval }}
 {{- end }}
+    metricRelabelings:
+    - sourceLabels: [ __name__ ]
+      regex: ^(go_goroutines|go_memstats_alloc_bytes|go_memstats_heap_alloc_bytes|go_memstats_heap_inuse_bytes|go_memstats_heap_sys_bytes|go_memstats_stack_inuse_bytes|process_cpu_seconds_total|process_max_fds|process_open_fds|process_resident_memory_bytes|process_start_time_seconds|process_virtual_memory_bytes)$
```

Comment:
`kyma-event-bus` dashboard does not have any data. It uses `total_published_messages` metrics. Do we need to include this here?